### ISSUE TITLE
Exception thrown in rider 2019.1.2 when running Nuke task from gutter icon

### ISSUE THREAD

**Issue by TCBroad** [nuke-build/resharper#15]
When clicking the icon for a nuke Target via the gutter icon, I get an IDE exception thrown and the Target doesn't execute. This happens with both Debug and Run options.

![image](https://user-images.githubusercontent.com/1906419/59455053-4c748580-8e0b-11e9-8bc5-b7ceeb6830c7.png)

I am using the latest version of the nuke plugin in Rider 2019.1.2 on windows 10. The solution in question has a lot of run configurations, if that makes any difference.

**Comment by matkoch** [comment]
I need your whole build implementation, at least the targets. Do you have more than a single build project in your solution? Or maybe similar named projects?

**Comment by matkoch** [comment]
It's probably this: https://github.com/nuke-build/resharper/blob/master/src/rider/main/kotlin/com/jetbrains/rider/plugins/nuke/RunConfigurationManager.kt#L68

And it would indicate that you have added the project multiple times.

**Comment by TCBroad** [comment]
I did have a run config called "_build" (the same name as the project) when i posted the issue. I have since removed it but the issue persists. There is only one project called _build in the solution but there are some other solution folders that start with @Build, though i wouldn't imagine they're being picked up by the code you linked.

I can try and send you the project tomorrow, it's got a lot of configurations in it and i will need to sanitise some of it.
